test/Turn-test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Turn = require('../src/Turn');
+const Card = require('../src/Card')
+
+describe('Turn', function() {
+
+    it('should be a function', function() {
+        const turn = new Turn();
+        expect(Turn).to.be.a('function');
+    })
+
+    it('should be an instance of Turn', function() {
+        const turn = new Turn();
+        expect(turn).to.be.an.instanceOf(Turn);
+    })
+
+    it('should store the user/s guess', function() {
+        const turn = new Turn('octopus');
+
+        expect(turn.guess).to.equal('octopus');
+    })
+
+    it('should store the card that is being answered')
```

Comment:
Nice work writing all of the it shoulds, and making a list of all the things the Turn class needs.